### ISSUE TITLE
Unable to Upgrade to Zikula 1.4.4 from 1.4.3

### ISSUE THREAD

**Issue by shefik** [zikula/core#3284]
| Q                 | A
| ----------------- | ---
| Zikula Version    | 1.4.4
| PHP Version       | 5.6.27

### Expected behavior

The upgrade page should display at `/upgrade`.

### Actual behavior

```( ! ) Fatal error: Uncaught exception 'JMS\I18nRoutingBundle\Exception\NotAcceptableLanguageException' with message 'The requested language "" was not available. Available languages: "en, en"' in /vendor/jms/i18n-routing-bundle/JMS/I18nRoutingBundle/Router/I18nRouter.php on line 236
( ! ) JMS\I18nRoutingBundle\Exception\NotAcceptableLanguageException: The requested language "" was not available. Available languages: "en, en" in /vendor/jms/i18n-routing-bundle/JMS/I18nRoutingBundle/Router/I18nRouter.php on line 236
```

### Steps to reproduce

Just try to upgrade from Zikula 1.4.3 to 1.4.4, and go to `/upgrade`.

**Comment by Guite** [comment]
> Just try to upgrade from Zikula 1.4.3 to 1.4.4, and go to /upgrade.

If that would be enough to reproduce it it would happen for all users. But there were several persons who tested the upgrade before the release happened. So there must be something specific to your environment. Do you have a default language other than English? If yes, do you use translated routes for that language? What else could be different? Are you able to provide a database dump?

**Comment by shefik** [comment]
I narrowed down the issue. Looks like there might a bug with the vendor code. I'm submitting a pull request to the vendor for my proposed fix. The conditional statement for populating the default locale needs to be moved.

**Comment by shefik** [comment]
I do not have any other language other than English, and I am not using translated routes.

**Comment by Guite** [comment]
> I'm submitting a pull request to the vendor for my proposed fix.

Nice! Please link it with this issue for reference.

**Comment by Guite** [comment]
https://github.com/schmittjoh/JMSI18nRoutingBundle/pull/199

**Comment by shefik** [comment]
My environment is MAMP PRO 4.0.6 on Mac.

**Comment by craigh** [comment]
this was fixed in #3396
